### DISCUSSION TITLE
[Bug] Don't use invite sniper, got me banned

### DISCUSSION THREAD

**Discussion by Maki-01** [marioparaschiv/nitro-sniper#111] [Show and tell]
I don't know how but I just turned it on to test it out, it joined one server and i left it right after.
15 seconds later, logged out everywhere :/ (It joined valorant's server, its parntnered and I heard that they have some sort of better protection than other servers)
Anyways,
Sharing this so someone could learn from my mistakes, 4 years old account lost :D
Only use it for alts and don't touch its default settings, timer is broken anyways.
Soo use it on alts mode not both mode like me dumbfu**...

**Comment by fucksophie** [comment]
Disable invitation setting. It is VERY VERY bad to use it. Do not enable it.

**Reply by Maki-01** [reply]
Ik, I actually got my account back, soo that's good. But don't use the invite sniper it is broken indeed.

**Reply by Serial-Comma** [reply]
damn lucky, used it on an alt with my phone number and they refused to give me back my account. Guess my phone number is permanently locked there ¯\_(ツ)_/¯

**Reply by Maki-01** [reply]
> damn lucky, used it on an alt with my phone number and they refused to give me back my account. Guess my phone number is permanently locked there ¯_(ツ)_/¯

Actually, you can just put yours number on another account after some amount of time... Not instantly though.

**Reply by yungcartionthachair** [reply]
> Disable invitation setting. It is VERY VERY bad to use it. Do not enable it.

bad to use as inherently broken or that discord is just picky with it joining servers? like would you say its cool to have it join 1-2 servers then pause for 48 hours?

**Reply by fucksophie** [reply]
> > Disable invitation setting. It is VERY VERY bad to use it. Do not enable it.
> 
> bad to use as inherently broken or that discord is just picky with it joining servers? like would you say its cool to have it join 1-2 servers then pause for 48 hours?

Just using it AT ALL doesn't work at the moment

**Comment by Serial-Comma** [comment]
Out of curiosity, why is it bad? I use it to automatically join those scam giveaway servers who have like a million affiliated servers at a rate of one server per hour with 120 seconds delay. My account also got banned. What makes it so obvious its a bot?

**Reply by InimicalPart** [reply]
Yeah, I wonder too, might investigate and return later

**Reply by Serial-Comma** [reply]
On another account (that isn't phone verified as well) I joined a ton of the same servers at a higher rate than the sniper and it didn't get banned. Something's definitely going on with the bot.
